scripts/i18n-helper.js

```diff
@@ -0,0 +1,72 @@
+export default class I18nHelper {
+  constructor() {}
+
+  /**
+   * If midi-qol module is present and active
+   *
+   * @readonly
+   * @memberof I18nHelper
+   */
+  get _usesMidiQol() {
```

Comment:
Private functions don't require JSdocs. Any non-private function would, like the i18n ones below

Reply:
Ok